.gitignore

```diff
@@ -1 +1 @@
-bin/*
```

Comment:
We shouldn't remove this line. The Makefile places output there.

Reply:
ty!